scene/animation/animation_blend_tree.cpp

```diff
@@ -384,9 +384,9 @@ void AnimationNodeAnimation::_bind_methods() {
 	ADD_PROPERTY(PropertyInfo(Variant::INT, "play_mode", PROPERTY_HINT_ENUM, "Forward,Backward"), "set_play_mode", "get_play_mode");
 	ADD_PROPERTY(PropertyInfo(Variant::BOOL, "advance_on_start"), "set_advance_on_start", "is_advance_on_start");
 	ADD_PROPERTY(PropertyInfo(Variant::BOOL, "use_custom_timeline"), "set_use_custom_timeline", "is_using_custom_timeline");
-	ADD_PROPERTY(PropertyInfo(Variant::FLOAT, "timeline_length", PROPERTY_HINT_RANGE, "0.001,60,0.001,or_greater,or_less,hide_slider,suffix:s"), "set_timeline_length", "get_timeline_length");
+	ADD_PROPERTY(PropertyInfo(Variant::FLOAT, "timeline_length", PROPERTY_HINT_RANGE, "0.001,60,0.001,or_greater,or_less,hide_control,suffix:s"), "set_timeline_length", "get_timeline_length");
```

Comment:
Here, there are projects that uses `hide_slider` hint string either while using the annotation `@export_range` or inside `_validate_property` method.

The complication comes from the approach taken, which could have been avoided by simply adding one more hint and a variable to control whether sliders are used for integer properties (with no effect on float properties). That way, we could also document that it’s possible to hide the up/down arrows for integer variables and show a slider instead using the `use_slider` hint string, and still hide the slider for both integer and float properties using `hide_slider`.

Reply:
I don't quite understand the problem here? As I said, "hide_slider" still works, so compatibility isn't broken.

The new hint to always use sliders is "prefer_slider", which can be overridden if used with "hide_control"/"hide_slider".

Reply:
Here's how it works:
- "prefer_slider": Makes integer values use the slider instead of the up-down arrows.
- "hide_control": Hides whatever is being used, either the slider or the arrows.
- "hide_slider": Same as above, but internally deprecated. Since its function is to hide not just the slider, but also the arrows, renaming "slider" to "control" makes more sense.